integration/commands_authentication_test.go

```diff
@@ -14,7 +14,29 @@
 
 package integration
 
-import "testing"
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/mongo-driver/bson"
+
+	"github.com/FerretDB/FerretDB/integration/setup"
+)
+
+func TestLogOut(t *testing.T) {
+	t.Helper()
+
+	t.Parallel()
+	ctx, collection := setup.Setup(t)
+
+	db := collection.Database()
+
+	var res bson.D
+	err := db.RunCommand(ctx, bson.D{{"logout", 1}}).Decode(&res)
```

Comment:
At the moment, change made by `conninfo.Get(ctx).SetAuth("", "")` is not obvious from the test. How about if we use `connectionStatus` command to check `authenticatedUsers` before and after `logout`? This way, we can check that the test user is authenticated before `logout`, then after `logout` the test user is no longer authenticated.

Let me know if you need any help :hugs:

Reply:
In our Taskfile.yml, we do TLS authentication for `task test-integration-pg` by setting `-target-tls`.
https://github.com/FerretDB/FerretDB/blob/2716382103d28323d6583a959d048e17eb880ed4/Taskfile.yml#L194

Our other integration tests like `task test-integration-mongodb` and `task test-integration-tigris` don't set TLS authentication. So those can be skipped for that test by 
```
setup.SkipForMongoDB(t, "tls is not enabled for mongodb backend")
setup.SkipForTigrisWithReason(t, "tls is not enabled for tigris backend")
```